components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher/src/test/java/org/wso2/carbon/apimgt/rest/api/publisher/common/SampleTestObjectCreator.java

```diff
@@ -0,0 +1,285 @@
+/*
```

Comment:
Yes until that we can use this way. When we create a common test module we can use it. When we create a separate test module actually we need to create a separate test jar and embedd all common things.